internal/command/views/state_locker.go

```diff
@@ -29,6 +33,8 @@ type StateLockerHuman struct {
 	view *View
 }
 
+// StateLockerHuman and StateLockerJSON share the same methods so we only have to check
+// whether StateLockerHuman conforms the interface
```

Comment:
I think it would be good if you could check both, we don't have a guarantee that someone won't come and add/remove functions in the future.